### DISCUSSION TITLE
Anyone know if you can get a single connector to plug onto the ESP32

### DISCUSSION THREAD

**Discussion by vaderag** [darthcloud/BlueRetro#41] [Q&A] [answered]
I'm looking for 19pin molex or similar to plug straight onto the ESP32 and then wire the ends as required... can't seem to find such a thing?
Is the only way to do this with individual jumper cables?

Planning to use something like this to avoid any soldering issues! 
https://www.amazon.co.uk/breakout-Solder-free-Adapter-Terminal-Connector/dp/B07DPDPYSS/

**Comment by darthcloud** [comment]
You means this https://www.amazon.ca/Screw-Terminal-Breakout-Module-ESP32-DevKitC/dp/B0896RKP8T ?

**Reply by vaderag** [reply]
That's pretty cool, but I was more looking for a cable 
![image](https://user-images.githubusercontent.com/17175988/121393245-13167800-c948-11eb-96fd-90462c55cf74.png)

Something like that, but with 19 connectors joined to plug straight on to either side

**Comment by darthcloud** [comment, marked as answer]
https://www.digikey.ca/short/vrbtjm7b
